app/src/lib/dispatcher/actions.ts

```diff
@@ -50,7 +50,13 @@ export interface IUpdateRepositoryPathAction {
   readonly path: string
 }
 
-export type Action = IGetAccountsAction | IGetRepositoriesAction |
-                     IAddRepositoriesAction | IUpdateGitHubRepositoryAction |
-                     IRemoveRepositoriesAction | IAddAccountAction | IRemoveAccountAction |
-                     IUpdateRepositoryMissingAction | IUpdateRepositoryPathAction
+export type Action =
+  | IGetAccountsAction
```

Comment:
😮 Turns out that’s [completely valid](http://www.typescriptlang.org/play/index.html#src=type%20foo%20%3D%20%7C%20string%20%7C%20number%20%7C%20boolean%0D%0Avar%20x%3A%20foo%20%3D%20true)